src/main/resources/archetype-resources/deployment/bin/build

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -ex
+
+REPO=074509403805.dkr.ecr.eu-west-1.amazonaws.com
+
+DIR=$( dirname $0 )
+
+cd ${DIR}/../..
+ROOT=`pwd`
+cd -
+
+DOCKER_PATH=${ROOT}/deployment/docker
+
+docker run --rm=true \
+  --user=$(id -u $USER):$(id -g $USER) \
+  -e HOME=/tmp \
+  -v ${ROOT}:/project \
+  -v ${HOME}/.m2/repository:/m2/repository \
+  ${REPO}/rebuy-base-image-builder-maven
```

Comment:
I think using a image name like [this](https://github.com/rebuy-de/digital-pick-app/blob/64bc7cd/deployment/bin/build#L9) would be nice, since it avoids race conditions between master and PR builds.